docs/docs/configuration/server.rst

```diff
@@ -229,6 +229,35 @@ timeout
     specify a timeout in seconds for blocking operations like the
     connection attempt.
 
+Mail
+----
+
+configure the format the mail notification uses in SMTP section.
```

Comment:
Please capitalize the first word.

s/uses in SMTP section/uses in the SMTP section/